.github/workflows/check.yml

```diff
@@ -7,6 +7,17 @@ on:
       - main
   # ..and any pull request.
   pull_request:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'warning'
+      tags:
+        description: 'Run Checks'
```

Comment:
```suggestion
      ref:
        description: 'The branch, tag or SHA to checkout (uses the default branch if empty)
```

Reply:
Added in b9d0655730838bd3ead19099d0e1f01751e91abf! Do you know of a way to test prior to merging?